projects/bp-gallery/src/hooks/bulk-operations.hook.tsx

```diff
@@ -32,10 +34,10 @@ const useBulkOperations = (parentCollection?: FlatCollection) => {
           );
         });
       },
-    },
+    } as BulkOperation,
```

Comment:
Why add the cast here and not to the other operations? Also, I think a type annotation (for the return type of the hook) instead of a cast is better, since I think the cast reduces type checks.

Reply:
Done